hummingbird/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+
+class UserProfile(models.Model):
+
+	name = models.CharField(max_length=40, null=True)
+	last_played = models.DateTimeField(null=True, blank=True)
+	song = models.FileField(upload_to='media/songs', null=True, blank=True)
+	length = models.IntegerField(default=10,
+				validators=[
+				MaxValueValidator(120),
+				MinValueValidator(1)
+				]
+			)
```

Comment:
nit: This parenthesis looks like it's hanging. I would reduce the indent on lines 12-16.